src/core/inlines.js

```diff
@@ -51,8 +52,21 @@ function inlineRFC2119Matches(matched) {
 function inlineRefMatches(matched) {
   // slices "[[[" at the beginning and "]]]" at the end
   const ref = matched.slice(3, -3).trim();
-  const nodeElement = hyperHTML`<a data-cite="${ref}"></a>`;
-  return nodeElement;
+  if (ref.startsWith("#")) {
+    if (!document.querySelector(ref)) {
+      const badReference = hyperHTML`
```

Comment:
Nit: Can we do this in one line?

Reply:
`showInlineError()` without actual DOM element is no useful, because the function is to give a link to the document.

Reply:
Somehow commented here because its same line 😁